docs/index.rst

```diff
@@ -12,6 +12,24 @@ The Roman Space Telescope Calibration Pipeline
    :align: center
    :alt: Nancy Roman Space Telescope
 
+Welcome to the documentation for the Roman calibration software,
+`romancal`. This package contains the Python software suite for the
```

Comment:
`romancal` is written as a link but is not actually linked.

Reply:
Fixed the link.